windows/ReactNativePicker/pch.h

```diff
@@ -1,6 +1,9 @@
 ﻿#pragma once
 
 #define NOMINMAX
+#define XAML_NAMESPACE L"Windows.UI.Xaml"
```

Comment:
if this could go into a header outside the specific picker control, that'd be great (not sure if we already have other controls in this repo?)

Reply:
This repo is picker only. Maybe we should add it somewhere in react-native-windows, perhaps Microsoft.ReactNative.Cxx.